inst/bs3compat/_declarations.scss

```diff
@@ -17,3 +17,9 @@ $selectize-color-item-border: if(is-css-variable($input-border-color), $border-c
 $selectize-color-dropdown: if(is-css-variable($dropdown-bg), $body-bg, $dropdown-bg) !default;
 $selectize-color-dropdown-border-top: rgba($input-border-color, 0.2) !default;
 $selectize-color-dropdown-item-create-text: rgba($selectize-color-text, 0.5) !default;
+@if variable-exists("input-btn-font-size") {
+  $selectize-font-size: $input-btn-font-size !default;
+} @else if variable-exists("font-size-base") {
+  $selectize-font-size: $font-size-base !default;
+}
```

Comment:
Let's make sure this is well tested ('m not sure this will change the behavior since the variable definition happens in a local scope)

Reply:
Tested locally and this worked as expected across Bootstrap versions (I think BS < 4 didn't have `$input-btn-font-size`).